crates/bevy_ecs/src/schedule/executor/multi_threaded.rs

```diff
@@ -444,7 +456,9 @@ impl MultiThreadedExecutor {
             #[cfg(feature = "trace")]
             let system_guard = system_span.enter();
             let res = std::panic::catch_unwind(AssertUnwindSafe(|| {
-                // SAFETY: access is compatible
+                // SAFETY:
+                // - Access: TODO.
```

Comment:
is this meant to stay as a TODO?

Reply:
Yes. The previous "access is compatible" comment is incorrect, since it's not supported by any invariants in this function. I'd rather be explicit that these invariants aren't fully fleshed out yet than pretend that they are. I have a real fix for this in #8292.